src/views/dashboard/asideBar/index.jsx

```diff
@@ -0,0 +1,65 @@
+import React from 'react';
+import { FaTachometerAlt } from 'react-icons/fa';
+import { FaBoxOpen } from 'react-icons/fa';
+import { FaChartBar } from 'react-icons/fa';
+import { FaFileAlt } from 'react-icons/fa';
+import { AiOutlineSetting, AiOutlineLogout } from 'react-icons/ai';
+import cart from '../../../assets/img/cart.png';
+import './style.scss';
+
+function AsideBar() {
+	return (
+		<div className='aside-bar'>
+			<div className="aside-bar-container">
+				<div className="cart-content">
+					<img src={cart} alt="" />
+					<h3>Amigos</h3>
+				</div>
+				<nav>
+					<ul>
+						<li>
+							<a href="">
+								<FaTachometerAlt className="fa" />
+								Dashboard
+							</a>
```

Comment:
Use Link from react-router-dom instead of html normal anchor tad, otherwise your web will reload upon clicking on the link which is not a react behavior

Reply:
sure